### PR TITLE
[MRG+1] DOC: Document `verts` in scatter

### LINKED ISSUE #6958: Document the verts kwarg to scatter
As of 2.0b3 the `verts` kwarg to `scatter` is not listed in the docstring parameter list.  It doesn't fall under the "other parameters: Collection properties" either.

### PR DESCRIPTION
**Pull request by tacaswell**
close #6958

### CODE REVIEW

**Comment by QuLogic** [comment]
So I have to ask, in what units? Pixels? Points? Or between 0 and 1 (and scaled by the size)?

**Comment by NelleV** [comment]
Hi @tacaswell 
If you don't plan on fixing @QuLogic comment any time soon, I suggest we merge this and someone can improve the documentation later.

**Comment by tacaswell** [comment]
thanks for the ping, updated and force-pushed.

File: lib/matplotlib/axes/_axes.py [hunk 1]
```diff
@@ -3811,8 +3811,7 @@ def scatter(self, x, y, s=None, c=None, marker=None, cmap=None, norm=None,
                 vmin=None, vmax=None, alpha=None, linewidths=None,
                 verts=None, edgecolors=None,
                 **kwargs):
-        """
-        Make a scatter plot of x vs y, where x and y are sequence like objects
+        """Make a scatter plot of x vs y, where x and y are sequence like objects
```
Comment (QuLogic) [inline lib/matplotlib/axes/_axes.py]: Not sure this should have changed.

File: lib/matplotlib/axes/_axes.py [hunk 2]
```diff
@@ -3861,6 +3860,11 @@ def scatter(self, x, y, s=None, c=None, marker=None, cmap=None, norm=None,
         linewidths : scalar or array_like, optional, default: None
             If None, defaults to (lines.linewidth,).
 
+        verts : sequence of (x, y), optional
+            If `marker` is None, these vertices will be used to
+            construct the marker.  The center of the marker is located
+            at (0,0) and the size is normalized.
```
Comment (QuLogic) [inline lib/matplotlib/axes/_axes.py]: I think I know, but normalized from what to what? Also, "the size" (as an unqualified term) could be confused with the `s` parameter.

File: lib/matplotlib/axes/_axes.py (hunk 1 above)
Comment (QuLogic) [inline lib/matplotlib/axes/_axes.py]: I'm pretty sure this will break something, though it may be the `dedent_interpd` that was removed...

Comment (NelleV) [inline lib/matplotlib/axes/_axes.py]: That indeed can break some indentation thing we have going on, but only if we are appending or interpolating this function.

Comment (NelleV) [inline lib/matplotlib/axes/_axes.py]: Note: the documentation build will fail if that is the case.

**Comment by tacaswell** [comment]
At the risk of kicking a hornets nest, re-worded the scatter summary/extended summary.

@efiring and/or @NelleV (sorry, memory failing) did some work to clarify the difference between plot and scatter, where did that end up landing?

**Comment by NelleV** [comment]
@tacaswell Excellent questions… I indeed had a pull request adding an element in the notes section about using plot instead of scatter…

**Comment by NelleV** [comment]
It is in the see also section: https://github.com/matplotlib/matplotlib/pull/7096

**Comment by NelleV** [comment]
I've got it locally in master, but I can't see it in your file.

**Comment by tacaswell** [comment]
Ah, great.  That explains why it was not showing up in my file on this branch (as it was forked before that PR was backported).

**Comment by QuLogic** [comment]
Were you going to rebase, or is this fine?

**Comment by tacaswell** [comment]
It's fine as-is, the changes are non-conflicting.

**Pull request merged**